MCDataProducts/inc/StepPointMC.hh

```diff
@@ -106,7 +106,9 @@ namespace mu2e {
       _volumeId(0),
       _totalEnergyDeposit(0.),
       _nonIonizingEnergyDeposit(0.),
+      _visibleEnergyDeposit(0.),
       _position(),
```

Comment:
My understanding is that the StepPointMC is a transition between a prePosition and postPosition. Should we rename position as prePosition since we now have a PostPosition?

Reply:
Although, technically, it is pre step point position, to be consistent we would need to rename the accessor, therefore changing the API. And, I think the schema evolution would be affected. Besides, it is the position which determines the volume id and other local properties, so I would rather not rename it.

Reply:
I will start an issue to discuss the long list of requests/suggestions for evolving StepPointMC.  I agree that it's badly named (I take credit for that!), takes up too much space and that we would be well served to define more targeted classes for many of our sensitive detector types.  It will be an intrusive change and needs to be carefully coordinated.